Candidates.txt

```diff
@@ -0,0 +1,3 @@
+Jovan Stosic jovan-stosic
```

Comment:
If you started with fresh directory you should have only your name here - so only one record
Where did you get others? If you think that you don't have a time to start with fresh new directory and do all exercises again, then remove that record.